libraries/vdf/include/vdf/sortition.hpp

```diff
@@ -27,8 +27,12 @@ class VdfSortition : public vrf_wrapper::VrfSortitionBase {
   explicit VdfSortition(bytes const& b);
   explicit VdfSortition(Json::Value const& json);
 
-  void computeVdfSolution(SortitionParams const& config, dev::bytes const& msg);
-  void cancelCompute();
+  // Only for testing
+  void computeVdfSolution(const SortitionParams& config, const bytes& msg);
```

Comment:
What is this function good for ? Why is not computeVdfSolutionCancellable used in tests ?